RussianFilter/sections/adservers.txt

```diff
@@ -2,6 +2,7 @@
 ! Содержит доменные имена, использующиеся рекламными сетями.
 !
 ||uezbshzpdcbb.info^$third-party
+||listonnw.pro^$third-party
```

Comment:
it seems bundle.js on this site can be blocked.

Reply:
Also missed `window.RESOURCE_`

Reply:
@zloyden @Alex-302 fixed